### DISCUSSION TITLE
Bug when using `Nav::extend`

### DISCUSSION THREAD

**Discussion by robdekort** [trendyminds/statamic-palette#5] [General]
I'm not sure where to open up an issue on this repo, so I'm gonna give it a shot here. Hope you don't mind. Great addon, thanks!

When you use custom nav items to extend the control panel like this:

```php
Nav::extend(function ($nav) {
    $nav->content('Pricing')
        ->icon('pro-ribbon');
});
```

Those nav items get loaded into Palette and this results in a confusing interface. They should probably be taken out. See the screenshots for what happens.

Best!

<img width="706" alt="Screenshot 2024-03-20 at 19 54 07" src="https://github.com/trendyminds/statamic-palette/assets/69107412/e6910573-ae72-4b95-b018-9bc8109d48ae">
<img width="701" alt="Screenshot 2024-03-20 at 19 53 59" src="https://github.com/trendyminds/statamic-palette/assets/69107412/e534264e-73c9-42f5-9431-1ae6d5282b1f">
<img width="705" alt="Screenshot 2024-03-20 at 19 53 50" src="https://github.com/trendyminds/statamic-palette/assets/69107412/bdc3bd10-1cff-4a7e-ad31-5be04d5f6cd5">

**Comment by aaronbushnell** [comment]
Hey @robdekort — just for my knowledge what do these items in the control panel nav do when you click on them normally?

**Comment by robdekort** [comment]
Heya, I add taxonomies/collections/navigations to them via the CP Nav feature. I could create those `Pricing` and `Guides` groups using that feature as well, but then they don't get an icon. Which is why I do it programatically.
<img width="470" alt="Screenshot 2024-03-28 at 08 25 45" src="https://github.com/trendyminds/statamic-palette/assets/69107412/44606baf-76e4-4f6a-b7c9-a64ed56b4539">
<img width="973" alt="Screenshot 2024-03-28 at 08 27 04" src="https://github.com/trendyminds/statamic-palette/assets/69107412/8c252846-d646-4e12-86c9-91dd2aff374b">
Does that help?

**Reply by aaronbushnell** [reply]
It does! So does the "Pricing" item not have a URL? A quick fix on my end might be to just exclude any items that lack a URL (otherwise there's nowhere for Palette to take you anyway)

**Reply by aaronbushnell** [reply]
If this is the case, 1.0.4 is out with this update!

**Reply by robdekort** [reply]
Hey Aaron. Thanks for your effort. They do have a URL, I manually link them to the first child. So Pricing goes to `cp/collections/tiers` and Guides goes to `/cp/collections/guides`. 

Sorry to inform you that the recent update still seems to include and duplicate them into the palette.
